sentry_sdk/integrations/django/middleware.py

```diff
@@ -102,11 +121,20 @@ def sentry_wrapped_method(*args, **kwargs):
 
         return old_method
 
-    class SentryWrappingMiddleware(object):
-        def __init__(self, *args, **kwargs):
-            # type: (*Any, **Any) -> None
-            self._inner = middleware(*args, **kwargs)
+    class SentryWrappingMiddleware(_asgi_mixin_factory(_check_middleware_span)):
+
+        async_capable = getattr(middleware, "async_capable", False)
+
+        def __init__(self, get_response=None, *args, **kwargs):
+            # type: (F, *Any, **Any) -> None
+            if get_response:
```

Comment:
When would this be `None`?

Reply:
This is for backwards compatibility. Older versions of Django only, should be deprecated in 4.0

Ref from django.utils.deprecation:
```python
# RemovedInDjango40Warning: when the deprecation ends, replace with:
    #   def __init__(self, get_response):
    def __init__(self, get_response=None):
```